resources/flows/covid-flow.js

```diff
@@ -153,14 +153,24 @@ export default [
 		requiresAnswer: [{ question: "thermometer", value: 11 }],
 		questionText: "TODO: Add numerical input",
```

Comment:
Could you also change the text here to "Select your temperature" or something equivalent